drop-transfer/src/service.rs

```diff
@@ -199,12 +204,23 @@ impl Service {
 
     pub async fn send_request(&mut self, xfer: crate::OutgoingTransfer) {
         let xfer = Arc::new(xfer);
+
+        self.state.moose.event_transfer_intent(xfer.info());
```

Comment:
`intent`? Is it a typo?

Reply:
intent is the correct name of the event, the word roughly means the wish or the intention of something, and this events describes just that - the intention to send an event, before any other action like connection is made